src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -118,13 +123,14 @@ class UniversalNavbar extends React.Component {
                   {LogoWhite({ className: 'universal-navbar-logo' })}
                 </NavLink>
                 <Spacer.H56 />
-                {LINKS.NAVLINKS.map((l) => (
-                  <div key={l.title + 'mobile'} style={{ marginBottom: 24 }}>
+                {LINKS.NAVLINKS.map((link) => (
+                  <div key={link.id} style={{ marginBottom: 24 }}>
                     <TitleXLarge.Sans.Regular400>
                       <NavLink
-                        key={l.title + 'mobile'}
-                        href={l.href}
-                        LinkComponent={LinkComponent}
+                        href={link.href}
+                        LinkComponent={
+                          l.href !== '/login/' ? LinkComponent : null
```

Comment:
Sorry, my coffee hasn't kicked in...can you explain how the `null` case would happen and result here?

Reply:
This is the NavLink component. 
It first checks for `LinkComponent` and since null is not truthy, we use anchor tags
```jsx
const NavLink = ({ href, LinkComponent, ...props }) => {
  if (LinkComponent) {
    return <LinkComponent to={href} {...props} />
  }
  return <a href={href} {...props} />
}
```

Reply:
Might be worth commenting but not a big deal and up to you. thanks for explaining